.github/workflows/daily.yml

```diff
@@ -0,0 +1,222 @@
+        run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+      - name: Install Superface CLI
+        run: brew install superfaceai/cli/superface
+      - run: superface --version
+      - run: superface whoami
+      - run: superface prepare https://github.com/lob/lob-openapi/blob/main/lob-api-public.yml lob
+      - name: Manually update provider
+        run: |
+          echo '{
+            "name": "lob",
+            "defaultService": "default",
+            "services": [
+              {
+                "id": "default",
+                "baseUrl": "https://api.lob.com"
+              }
+            ],
+            "securitySchemes": [
+              {
+                "id": "lobBasicAuth",
+                "type": "http",
+                "scheme": "basic"
+              }
+            ],
+            "parameters": []
+          }' > superface/lob.provider.json
+      - run: superface new lob "add a new address" address-management/new-address
+      - run: superface map lob address-management/new-address
+      - run: superface map lob address-management/new-address python
+      - run: |
+          sed -i -s "s/assetsPath.*/assetsPath: '\.'/" superface/*.mjs
+      - run: |
+          sed -i -s 's/assets_path.*/assets_path = "\."/' superface/*.py
+      - name: Upload superface folder
+        uses: actions/upload-artifact@v3
+        with:
+          name: superface-folder
+          path: | 
+            superface/*.profile
+            superface/*.map.js
+            superface/*.provider.json
+            superface/*.mjs
+            superface/*.py
+            superface/package.json
+            superface/requirements.txt
+      - name: Report to Slack
+        if: failure()
+        uses: slackapi/slack-github-action@v1
+        with:
+          channel-id: ${{ secrets.SLACK_CHANNEL }}
+          slack-message: "Superface Daily: Install CLI on Linux Failed"
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+  perform_nodejs:
+    name: Perform Node.js 
+    needs: [create_integration]
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [18, 20]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      LOB_USERNAME: ${{ secrets.LOB_TOKEN }}
+      LOB_PASSWORD: ""
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Download superface folder
+        uses: actions/download-artifact@v3
+        with:
+          name: superface-folder
+          path: superface
+      - name: Install Dependencies
+        working-directory: superface
+        run: npm install
+      - name: Perform
+        working-directory: superface
+        run: node --experimental-wasi-unstable-preview1 --no-warnings address-management.new-address.lob.mjs | tee result.txt
```

Comment:
Maybe we could use `sf execute` to test it 🤷🏻‍♂️

Reply:
I am running it directly, because in this step CLI isn't installed. But as I today learned npm and node is on every os runner, I guess I can add it.